src/ServiceProvider.php

```diff
@@ -88,4 +89,17 @@ private function registerViewComposers(Factory $view): void
     {
         $view->composer('coreui::master', CoreUIComposer::class);
     }
+
+    private function loadTranslations()
```

Comment:
Nitpicking: it's called `loadTranslations` while it both `loads` and `publishes`. I did the same with `publishConfig()`, but maybe there's a nicer way to fix this. Good for now.